LayoutTests/accessibility/math-foreign-content.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE HTML>
+<!DOCTYPE HTML><!-- webkit-test-runner [ CoreMathMLEnabled=false ] -->
```

Comment:
Can we rewrite the test to make it work independently of the flag?

Does any of the following options work:
- remove the `csymbol` so `annotation-xml` becomes the first child and is rendered.
- add some CSS rules to explicitly make `annotation-xml` visible (probably only works if MathML Core is enabled).
- use `<math id="math"><mtext><svg height="30"> width="75">...</svg></mtext></math>` instead.